Firestore/core/src/firebase/firestore/model/snapshot_version.h

```diff
@@ -44,6 +44,9 @@ class SnapshotVersion {
   static const SnapshotVersion& None();
 
 #if defined(__OBJC__)
+  SnapshotVersion() : timestamp_{} {
```

Comment:
You don't need to explicitly initialize `timestamp_` here.

Reply:
done